Используйте определение условной вероятности.
P A|B =P(AB)/P(B)

In [77]:
let60 = 0.5
let80 = 0.2

let80/let60

0.4

Искомая вероятность — сумма вероятностей всех ячеек, отмеченных плюсом. Поскольку австралийцы выбираются независимо, в каждой ячейке вероятность равна произведению вероятностей соответствующих строки и столбца.

In [78]:
import numpy as np

p1=0.49
p2=0.38
p3=0.1
p4=0.03

p=[p1,p2,p3,p4]
print p

mat=np.array([
    [1,1,1,1],
    [0,1,0,1],
    [0,0,1,1],
    [0,0,0,1]
])

print mat

matrix = np.zeros((4,4))

for x,i in enumerate(mat):
    for y,j in enumerate(i):
        if j>0:
            matrix[x][y] = p[x]*p[y]

print matrix
print matrix.sum()


[0.49, 0.38, 0.1, 0.03]
[[1 1 1 1]
 [0 1 0 1]
 [0 0 1 1]
 [0 0 0 1]]
[[ 0.2401  0.1862  0.049   0.0147]
 [ 0.      0.1444  0.      0.0114]
 [ 0.      0.      0.01    0.003 ]
 [ 0.      0.      0.      0.0009]]
0.6597


In [25]:
tur=0.6
tur2=0.1

indi=0.4
indi2=0.15

x= tur2*tur
print x
y= indi2*indi
print y
xy = x+y

round(xy,2)

0.06
0.06


0.12

P(B)=PBAP(A)+PBA¯P(A¯)=0.1⋅0.6+0.15⋅0.4=0.12.

In [29]:
print x/xy

0.5



По формуле Байеса P(A|B)=P(B|A)P(A)/P(B)=0.1⋅0.6/0.12=0.5.

In [89]:
pop=0.064
kot=0.037

x= pop+kot
print x

y=x-kot
print y


0.101
0.064


p=0.064, если все, кто читает pop, читают и kot  , P (AB) = P (A)

p=0.101, если аудитории этих журналов не пересекаются.